### PR TITLE
Fix grammatical error

### PR DESCRIPTION
**Pull request by teamotitan**
Fix small grammatical error

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Action     | Format, Examples
 **Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
 **Edit** : edits a student in UGTeach | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/SCHEDULE] [s/SUBJECT] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`<br> e.g.,`edit 2 paid/1200.00 owed/0`
 **Delete** : deletes a student in UGTeach | `delete INDEX`<br> e.g., `delete 3`
-**Find** : finds a student in UGTeach with part of name matching one of keywords, **and** their schedule falling on one of the days specified | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
+**Find** : Finds students whose names contain any of the given keywords *and* their tuition day contains any of the given days | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex John d/Friday`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:75]: Maybe change tuition day to schedule

**Review by Soumyadip-cmd — COMMENTED** [review]
Some considerations

File: docs/UserGuide.md (hunk 1 above)
Comment (teamotitan) [reply docs/UserGuide.md:75]: ok ty

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Action     | Format, Examples
 **Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
 **Edit** : edits a student in UGTeach | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/SCHEDULE] [s/SUBJECT] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`<br> e.g.,`edit 2 paid/1200.00 owed/0`
 **Delete** : deletes a student in UGTeach | `delete INDEX`<br> e.g., `delete 3`
-**Find** : finds a student in UGTeach with part of name matching one of keywords, **and** their schedule falling on one of the days specified | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
+**Find** : Finds students whose names contain any of the given keywords *and* their schedule contain any of the given days | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex John d/Friday`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:75]: Sorry but I think should be schedules.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -259,7 +259,7 @@ Deletes the specified student from UGTeach.
 
 ### Finding students' information: `find`
 
-Finds students whose names contain any of the given keywords *and* their tuition day contains any of the given days.
+Finds students whose names contain any of the given keywords *and* their tuition day contain any of the given days.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:262]: Can change herre too thanks

**Pull request merged**